assets/messages.yml

```diff
@@ -713,6 +713,19 @@ messages:
         This is a technical support issue; this site is for bug reports only. We do not have the resources to provide you with technical support.
         Please check *MCL-6550* for help with your issue. If you still need assistance after that, please contact *[Community Support|https://discord.gg/58Sxm23]* and refer to this ticket (e.g. "I have reported this issue on Mojira as %project_id%-_{color:gray}XYZ{color}_.").
 
+        %quick_links%
+      fillname: []
+  legacy-launcher-version:
+    - project: mcl
+      name: Legacy launcher version
+      shortcut: leg
+      message: |-
+        You have selected the version "1.6.93 (legacy)" as affected version for your report. Nowadays most users are using the new native launcher.
+        If the launcher you are using has a big green "PLAY" button for starting the game, then you are using the new launcher. To find out the version you are using, please click on "Settings" in the lower left corner, then open the "About" tab. Right below "Launcher" it shows the version of the launcher.
+        Please update the affected versions of your report by removing "1.6.93 (legacy)" and choosing the correct version.
+
+        In case you are actually using the legacy launcher, consider switching to the new native launcher. It can be downloaded [here|https://www.minecraft.net/download/alternative].
```

Comment:
I chose https://www.minecraft.net/download/alternative here because that only lists the download links for the Java launcher.

https://www.minecraft.net/download unfortunately mixes Java and Bedrock without clearly differentiating them, so that would probably only be confusing.